apps/showcase/src/assets/trainings/sdk/steps/angular-integration/instructions.md

```diff
@@ -0,0 +1,40 @@
+When dealing with an Angular project, you need to ensure that your `ApiClient` will be shared accross
+your application. The Otter framework provides the `ApiManager` service to manage your API collection.
+
+### Objective
+- Leverage the `ApiManager` service to access two different clients to retrieve the list of available pets and submit an order for the first pet returned.
+- Add a plugin to the `OrderApi` to log each time a call is sent.
+
+### Prerequisite
+- The package `@o3r/apis-manager` needs to be installed in the project with `npm install @o3r/apis-manager` (which has already been done for you).
+
+### Exercise
+As you can see in the `app.config.ts` file, a plugin `RequestLogPlugin` has been created which displays an alert box when the API receives a request.
+
+Integrate the `ApiManagerModule` in your `ApplicationConfig` and configure it to use the `RequestLogPlugin` in the `StoreApi`.
+You can inspire yourself with the following lines:
+
+```typescript
+// Default configuration for all the APIs defined in the ApiManager
+const apiConfig: ApiClient = new ApiFetchClient(
+  {
+    // Properties of ApiFetchClient
```

Comment:
it would be nice to also have a plugin in the default config to see if the plugins are merged or replaced when overriding the config

Reply:
Could be done in a dedicated step about plugins no ?

Reply:
I can use the mock plugin that they should have created in the previous step